sdk/src/rent.rs

```diff
@@ -10,7 +10,7 @@ pub struct Rent {
     pub exemption_threshold: f64,
 
     // What portion of collected rent are to be destroyed, percentage-wise
-    pub burn_percent: u8,
+    burn_percent: u8,
```

Comment:
Why de-pub and add `get_burn_percentage()`, making `burn_percent` unlike the other two fields?

Reply:
@mvines It was done to make sure, that it cannot be modified arbitrary.

Reply:
The other two field would have that same concern though, and the downside of removing pub from only one is inconsistency and possible confusion.   I'd restore the `pub` here, and if we want to "clean up" the entire struct then do that as a separate PR (IMO it's not worth it though)